jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/PropertiesUtil.java

```diff
@@ -39,4 +39,20 @@ public static Properties getPropertiesFromResource(URL resource) {
     }
     return ret;
   }
+
+  /**
+   * Return first Non Null set property from a set of provided properties
+   *
+   * @param properties {@link Properties} in a given project
+   * @param keys an array of property key values to find
+   * @return a string which is first non null value found for the provided list
+   */
+  public static String getValueFromProperties(Properties properties, String[] keys) {
+    for (String property : keys) {
+      if (properties.containsKey(property)) {
+        return properties.get(property).toString();
```

Comment:
It might make sense to check if the property is not blank either

(get property, check if not null, check if not blank, return)